lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -147,41 +223,63 @@ module.exports = {
   implementation: EventTargetImpl
 };
 
-function invokeEventListeners(listeners, target, eventImpl) {
-  const wrapper = idlUtils.wrapperForImpl(target);
-  const document = target._ownerDocument || (wrapper && (wrapper._document || wrapper._ownerDocument));
-  // Will be falsy for windows that have closed
-  if (!document) {
-    return;
+// https://dom.spec.whatwg.org/#concept-event-listener-invoke
+function invokeEventListeners(tuple, eventImpl /* , legacyOutputDidListenersThrowFlag */) {
```

Comment:
Why is the flag commented out, here and below? We should leave a comment explaining.